glustolibs-gluster/glustolibs/gluster/volume_ops.py

```diff
@@ -817,7 +817,7 @@ def get_volume_list(mnode):
     root = etree.XML(out)
     vol_list = []
     for volume in root.findall("volList"):
-        for elem in volume.getchildren():
+        for elem in list(volume):
```

Comment:
Since `volume` is being used as a iterable, i guess there's no need for turning it into `list`.
Can you try removing `list` once?

Sooner or later we'll be hitting this issue in other parts of the code as well, would be good if you can change them as well? https://github.com/gluster/glusto-tests/search?q=getchildren

Reply:
It seems to be working fine without `list`. I'll change other parts of the code as well.